data/constants.js

```diff
@@ -29,6 +29,7 @@ const VAULT_CATEGORIES_IDS = {
   IDLE: 'IDLE',
   INACTIVE_POLYGON: 'INACTIVE_POLYGON',
   GENOMES: 'GENOMES',
+  MSTABLE: 'mStable',
```

Comment:
Sorry, this is not what I meant. It needs to be `MSTABLE: 'MSTABLE'` for VAULT_CATEGORIES_IDS, ESTIMATED_APY_TYPES and TRADING_APY_TYPES. Next to that you need to add an input to VAULT_CATEGORIES_NAMES which is `MSTABLE: 'mStable'`